File: KarateKlub/KarateKlub.Api/Models/TakmicenjaPregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{
    public class TakmicenjaPregledVM
    {
        public class Row {
            public int id;
            public string nazivTakmicenja;
            public DateTime datumOdrzavanjaTakmicenja;
            public string mjestoOdrzavanjaTakmicenja;
            public string organizatorTakmicenja;
            public int savezId;
            public string nazivSaveza;
        }
        public List<Row> rows;
    }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_trener/model/TakmicenjaPregledVM.java
package com.example.x.karateklub.area_trener.model;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class TakmicenjaPregledVM implements Serializable {
    public  static class Row implements  Serializable{
        public int id;
        public String nazivTakmicenja;
        public Date datumOdrzavanjaTakmicenja;
        public String mjestoOdrzavanjaTakmicenja;
        public String organizatorTakmicenja;
        public int savezId;
        public String nazivSaveza;

    }
    public List<Row> rows;
}
